server/core/store.go

```diff
@@ -251,9 +251,16 @@ func (s *StoreInfo) GetLastHeartbeatTS() time.Time {
 const minWeight = 1e-6
 const maxScore = 1024 * 1024 * 1024
 
-// LeaderScore returns the store's leader score: leaderSize / leaderWeight.
-func (s *StoreInfo) LeaderScore(delta int64) float64 {
-	return float64(s.GetLeaderSize()+delta) / math.Max(s.GetLeaderWeight(), minWeight)
+// LeaderScore returns the store's leader score.
+func (s *StoreInfo) LeaderScore(kind LeaderScheduleKind, delta int64) float64 {
+	switch kind {
+	case ScheduleLeaderBySize:
+		return float64(s.GetLeaderSize()+delta) / math.Max(s.GetLeaderWeight(), minWeight)
+	case ScheduleLeaderByCount:
+		return float64(int64(s.GetLeaderCount())+delta) / math.Max(s.GetLeaderWeight(), minWeight)
+	default:
```

Comment:
I don't think it's appropriate to return 0 here?

Reply:
There are only `ScheduleLeaderBySize` and `ScheduleLeaderByCount`